apk-requirements.txt

```diff
@@ -0,0 +1,5 @@
+bash
+curl
+libc6-compat
+xz-dev
+libressl-dev
```

Comment:
I am sorry if [this](https://github.com/mendersoftware/mender-ci-workflows/pull/1#discussion_r1003235298) wasn't clear - the `apk-requirements.txt` file(s) need to be in `mender-artifact` and `mender-cli` repositories, so that they are versioned correctly.

Again, the intention is to follow encapsulation principle: let the repositories evolve their dependencies without having to change where this repositories are used.

Reply:
Got it 👍

Reply:
I don't think we are understanding each other.

My suggestion is to _move_ the `apk-requirements.txt` to the source repos (mender-artifact, mender-cli) but not to have now the requirements spread out (like partially in the build repo, partially in the source repos).

So either we have everything here or we move them (I prefer this second option and I have reason it above). These bash, curl and libressl requirements I assume are for `mender-cli`, so we should move them there.

Reply:
Okey, I get it now. So `curl` and `bash` are for generic CI operations (good, makes sense) and `libressl` was there for no reason :sweat_smile: 

Looks good now! Thanks.